package/network/services/dnsmasq/files/dhcp.conf

```diff
@@ -15,7 +15,7 @@ config dnsmasq
 	option leasefile	'/tmp/dhcp.leases'
 	option resolvfile	'/tmp/resolv.conf.auto'
 	#list server		'/mycompany.local/1.2.3.4'
-	#option nonwildcard	1
+	option nonwildcard	1 # bind to & keep track of interfaces
```

Comment:
Most people either backup or put their favorite UCI in the build `files/etc/config` so this may not be adopted. I agree with the forced shift however, because its 99% transparent to most users, but useful for us to build on for future features. Might I suggest further eliminating this option in dnsmasq.init, and using new option `bind_dynamic`. Also, make dnsmasq.init assume a 1 if not included.